index.html

```diff
@@ -1,4 +1,4 @@
-<!DOCTYPE html>
+﻿<!DOCTYPE html>
```

Comment:
What are the changes you made here? @suhailsinghbains can you check what is it , I am not able to check what's the actually diff b/w these lines

Reply:
GitHub is high i think :wink:

Reply:
[Good read](https://stackoverflow.com/questions/11452675/git-diff-shows-the-same-line-as-both-deleted-and-added)
Probably some whitespace has been replaced with some other whitespace. May be due to replacing <space> with a <tab> or vice-versa, cannot comment much though.

Reply:
> 
> 
> GitHub is high i think 😉

Interesting and fascinated to see this, I have never seen this ever since I started ;P